api_specs/v2/component_analyses.yaml

```diff
@@ -8,10 +8,78 @@ servers:
 security:
 - tokenAuth: []
 paths:
+  /component-analyses/:
+    post:
+      tags:
+        - CA batch call
+      summary: Get security and other details about a software component and recommendations to change to a more secured version in a batched manner.
+      parameters:
+        - name: uuid
+          in: header
+          description: User Identification uuid
+          required: false
+          schema:
+            type: string
+      requestBody:
+        description: CA Batch Request
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                ecosystem:
+                  type: string
+                  example: pypi
+                payload:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      package:
+                        type: string
+                        example: django
+                      version:
+                        type: string
+                        example: 1.1
+              example:
+                ecosystem: "pypi"
+                payload:
+                  - package: "django"
+                    version : "1.1"
+                  - package: "requests"
+                    version : "2.1"
+
+      responses:
+        200:
```

Comment:
What if the given epvs are all unknown? Any distinct error code would be sent or always 200?

Reply:
Ack.! Added More Response Codes